### ISSUE TITLE
Community rules suggestion, to reduce noise on PRs and github issues with "My engine is not working/starting/running" type of issues

### ISSUE THREAD

**Issue by avirtopeanu-ionos** [ange-yaghi/engine-sim#144]
Hi, as this sim garnered a huge following, it makes sense to implement some community rules/guidelines on GitHub to reduce some of the irrelevant noise.

## Problems: ##

- Noise on PRs, comments that do not add any value to the PR but just generate more notifications and simply take more time to scroll through the PR's history. For example of PR noise, see [this PR's comments](https://github.com/ange-yaghi/engine-sim/pull/140)

- Git issues such as "My engine is not working", "My engine is not starting/running", "My .mr script has bad cam timings" etc. at most signify a need to have some documentation on how to design a proper engine, but they do not really add any helpful information.

### Suggestion for noisy PRs: ###

Add a "Community Note" PR template to remind viewers/commenters to not pollute the PR history:

```
Please vote on this pull request by adding a +1 to the original pull request comment to help the community and maintainers prioritize this request
Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
```

### Suggestions for reducing issue posts: ### 

- Add a "Engine problem guidelines" master thread as a github issue, or/and
- Add a wiki page where the community can contribute, or/and
- Simply suggest that these type of questions/issues/bug reports be forwarded to the "Support" text channel on the community Discord server, or/and
- Add a "label" for GitHub issues such as `Support request` to enable filtering these types of issues.

**Comment by sujaldev** [comment]
Github discussions would be much better in my opinion compared to discord, less chat here.

**Comment by Jckf** [comment]
I agree with the overall sentiment here, but have some comments nonetheless
 * We already have a support label :)
 * Splitting the community across multiple platforms is never a good idea. Discussions should either happen only on Discord, or only on Github. If we are to use both, then there **must** be very clear guidelines on what type of discussion belongs where.
